tests/integration/users/test_views.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from openslides.utils.test import TestCase
+
+
+class TestWhoAmIView(TestCase):
+    url = '/users/whoami/'
+
+    def test_get_anonymous(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content, b'{"user_id": null}')
```

Comment:
I use a subclass of the Django Rest Framework APIView know, but we should not use ```.data``` in integration tests. ```.data``` is not the result of the view. See: http://www.django-rest-framework.org/api-guide/responses/#data

I see no problem to use the default encoding of the test file. If we use request.content.decode() it would be exactly the same but with the disadvantages that not the result is tested, but an altered version of it.